### ISSUE TITLE
sound: create documentation for text-to-speech

### ISSUE THREAD

**Issue by jwschroeder21** [uchicago-cs/chiventure#1577]
In this issue, we will create documentation for the text-to-speech feature in the sandbox module. We will create a README file that explains how to implement and use this feature. Hopefully, when the pull request is merged, other teams will be able to understand this feature and use it in their implementations.

Assignees: Rhea, Victor, Jake

**Comment by jwschroeder21** [comment]
Update: we are all in a Campus North study room playing with the text-to-speech feature. We are putting song lyrics in it and playing "super bass" by Nicki Minaj.

**Comment by jwschroeder21** [comment]
Update: linked Rhea's pull request for 'added documentation for the text to speech feature.' Waiting on reviews by Elizabeth and Jason before closing statement.

**Comment by jwschroeder21** [comment]
Closing statement:

In this issue, we created documentation for the text-to-speech feature in the sandbox module. We created a README file that explains how to implement and use this feature, which will help future teams. The pull request has been merged and now we can work on further developing the text to speech feature. Once we formally integrate text to speech, we will create a new issue that covers documentation in the main sound README.
